packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -435,4 +435,30 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
 
     Terra.it.matchesScreenshot('after offset change', { selector: '#root' });
   });
+
+  describe('onBlur (CST to CDT)', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-dst-blur');
+      browser.click('input[name="terra-time-minute-input"]');
```

Comment:
Should call `Terra.hideInputCaret('input[name="terra-time-minute-input"]');` to hide the input caret?

Reply:
Since I am not taking a screenshot immediately after clicking into the minute input field and instead I am tabbing out (next line) for the Time Clarification modal to appear, I think I shall keep this as-is.